datarobot_batch_scoring/network.py

```diff
@@ -19,12 +21,12 @@ class Network(object):
     def __init__(self, concurrency, timeout):
         self._executor = ThreadPoolExecutor(concurrency)
         self._timeout = timeout
+        self.session = requests.Session()
```

Comment:
we might watch out here -- Session appears to make no statements on thread-safety 

just found this note: http://stackoverflow.com/questions/18188044/is-the-session-object-from-pythons-requests-library-thread-safe